migrations/000001_create_tags_templates_fields_table.up.sql

```diff
@@ -0,0 +1,38 @@
+BEGIN;
```

Comment:
@mabdh Since these migrations are specific to postgres. They should better be placed in store/postgres package only.

Reply:
Yes it does make sense to store it inside postgres libraries. Will update it.

Reply:
@ravisuhag I have moved the folder and also update the migration with `go:embed` instead of source files directly.